src/api/app/components/watched_items_list_component.html.haml

```diff
@@ -11,8 +11,6 @@
           %i.fas.fa-cubes.mr-1
           #{item}
       - when 'BsRequest'
-        = link_to(request_show_path(number: item.number), class: 'text-word-break-all') do
-          = image_tag('icons/request-icon.svg', height: 18, class: 'mr-1 color-inverted')
-          Request ##{item.number}
+        = helpers.descriptive_request_link(item)
```

Comment:
What are you gaining by moving those three lines of haml into a helper? I know what `link_to` and `image_tag` etc. do. I have to look up what this helper does....

Reply:
I would also prefer: icon number type. Like 

![request-list](https://user-images.githubusercontent.com/514785/165296451-61ab2f04-a644-42d6-8bf0-1796831aa6a4.png)

Reply:
I no longer rely on a helper and the list displays icon, number and type as suggested.